intbot/core/tasks.py

```diff
@@ -43,22 +47,24 @@ def process_github_webhook(wh: Webhook):
         raise ValueError("Incorrect wh.source = {wh.source}")
 
     try:
-        message, event_action = parse_github_webhook(
-            headers=wh.meta, content=wh.content
-        )
+        wh = prep_github_webhook(wh)
     except ValueError as e:
         # Downgrading to info because it's most likely event not supported
         logger.info(f"Not processing Github Webhook {wh.uuid}: {e}")
         return
 
-    # NOTE WHERE SHOULD WE GET THE CHANNEL ID FROM?
+    parsed = parse_github_webhook(wh)
+    channel = discord_channel_router(wh)
+
     DiscordMessage.objects.create(
-        channel_id=settings.DISCORD_TEST_CHANNEL_ID,
-        channel_name=settings.DISCORD_TEST_CHANNEL_NAME,
-        content=f"GitHub: {message}",
+        channel_id=channel.channel_id,
+        channel_name=channel.channel_name,
+        # channel_id=settings.DISCORD_TEST_CHANNEL_ID,
```

Comment:
why is this commented out? if it's not needd, let's remove it

Reply:
Fixed